### PR TITLE
2323 tables

### PR DESCRIPTION
**Pull request by vsevolod-adrianov**
add unit angular table tests

### CODE REVIEW

File: jdi-light-angular-tests/src/main/java/io/github/com/pages/AngularPage.java [hunk 1]
```diff
@@ -6,32 +6,7 @@
...
-import io.github.com.pages.sections.SlideToggleSection;
-import io.github.com.pages.sections.SliderSection;
-import io.github.com.pages.sections.SnackbarSection;
-import io.github.com.pages.sections.SpinnerSection;
-import io.github.com.pages.sections.TabGroupSection;
-import io.github.com.pages.sections.ToolbarSection;
-import io.github.com.pages.sections.TooltipSection;
-import io.github.com.pages.sections.ExpansionPanelSection;
-import io.github.com.pages.sections.FormFieldsSection;
+import io.github.com.pages.sections.*;
```
Comment (gbuh) [inline jdi-light-angular-tests/src/main/java/io/github/com/pages/AngularPage.java:9]: Could you please return previous imports condition without asterisks.

**Comment by mf-code** [comment]
[6193] - fvkOBBqWABK3

**Comment by mf-code** [comment]
[6193] - h0wa11ppVYIN

**Comment by mf-code** [comment]
[6193] - cV3qwrAhZeVi

**Comment by mf-code** [comment]
[6193] - i0xPJCsqwSmU

File: jdi-light-angular-tests/src/main/java/io/github/com/pages/sections/TableFilteringSection.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+import com.epam.jdi.light.elements.complex.table.Table;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.JTable;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.ui.html.elements.common.TextField;
+
+public class TableFilteringSection {
+    @UI("#mat-input-84")
+    public static TextField filter;
+
+    @JTable(root = "table-filtering-example > table")
```
Comment (gbuh) [inline jdi-light-angular-tests/src/main/java/io/github/com/pages/sections/TableFilteringSection.java:12]: In the UI locator, the text "> " is optional.

File: jdi-light-angular-tests/src/main/java/io/github/com/pages/sections/TableFilteringSection.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
+package io.github.com.pages.sections;
+
+import com.epam.jdi.light.elements.complex.table.Table;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.JTable;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.ui.html.elements.common.TextField;
+
+public class TableFilteringSection {
+    @UI("#mat-input-84")
```
Comment (gbuh) [inline jdi-light-angular-tests/src/main/java/io/github/com/pages/sections/TableFilteringSection.java:9]: This identifier is automatically generated by the angular framework. Is another locator possible here?

File: jdi-light-angular-tests/src/main/java/io/github/com/pages/sections/TableOverviewSection.java [hunk 4]
```diff
@@ -0,0 +1,18 @@
+package io.github.com.pages.sections;
+
+import com.epam.jdi.light.angular.elements.complex.Paginator;
+import com.epam.jdi.light.elements.complex.table.Table;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.JTable;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.ui.html.elements.common.TextField;
+
+public class TableOverviewSection {
+    @UI("#mat-input-85")
```
Comment (gbuh) [inline jdi-light-angular-tests/src/main/java/io/github/com/pages/sections/TableOverviewSection.java:10]: same

File: jdi-light-angular-tests/src/main/java/io/github/com/pages/sections/TableOverviewSection.java [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    public static TextField overviewFilter;
+
+    @JTable(root = "table-overview-example>div>table")
```
Comment (gbuh) [inline jdi-light-angular-tests/src/main/java/io/github/com/pages/sections/TableOverviewSection.java:13]: same

File: jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/unit/TableUnitTests.java [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+        filteringTable.is().shown();
+        assertTrue(filteringTable.getColumn(1).size()<primaryTableRows);
+    }
+
+    @Test
+    public void footerTablesTest() {
+        tableWithFooter.show();
+        String[] total = tableWithFooter.footerUI().getValue().split(" ",2);
+        List<String> cost = tableWithFooter.getColumn(2).getValuesFast();
+        assertEquals(cost.get(cost.size()-1),total[1]);
```
Comment (gbuh) [inline jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/unit/TableUnitTests.java:90]: Please format the file before pushing to the repo.

File: jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/unit/TableUnitTests.java [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+import static io.github.com.pages.sections.TableOverviewSection.*;
+import static io.github.com.pages.sections.TableSection.*;
+import static io.github.epam.site.steps.States.shouldBeLoggedIn;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotEquals;
+import static org.testng.AssertJUnit.assertTrue;
+
+public class TableUnitTests extends TestsInit {
+
+    final static String[] chemElements = {"Hydrogen","Helium","Lithium","Beryllium","Boron","Carbon","Nitrogen"};
```
Comment (gbuh) [inline jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/unit/TableUnitTests.java:22]: If this variable is only uses in this class, please add private modifier to it.

**Comment by mf-code** [comment]
[6193] - cWxyxJeOUpzT

File: jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/unit/TableUnitTests.java [hunk 8]
```diff
@@ -0,0 +1,104 @@
+package io.github.epam.angular.tests.unit;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+import static io.github.com.StaticSite.angularPage;
+import static io.github.com.pages.sections.TableDynamicSection.*;
```
Comment (gbuh) [inline jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/unit/TableUnitTests.java:10]: Please avoid asterisks in the import statement.

**Comment by mf-code** [comment]
[6193] - nrs08yL0QQND

**Comment by mf-code** [comment]
[6193] - n7THnVCWmRFp

**Comment by mf-code** [comment]
[6193] - qK2BPcPMnlFL

**Comment by mf-code** [comment]
[6193] - r3gl7s0uri1Q

**Comment by mf-code** [comment]
[6193] - TPkgVxQBotCi

**Comment by mf-code** [comment]
[6193] - PsGP6Wu9i8Fe

**Comment by mf-code** [comment]
[6193] - YTjM02NI6Sng

**Comment by mf-code** [comment]
[6193] - FN6ydqhSQLqo

**Comment by mf-code** [comment]
[6193] - 1Ehphh8k4mdU

**Comment by mf-code** [comment]
[6193] - MLBccF5yMuDJ

**Comment by mf-code** [comment]
[6193] - kPEKkOjxP09b

**Comment by mf-code** [comment]
[6193] - lEefclJwI07t

**Comment by mf-code** [comment]
[6193] - DlhH0Y6CnL3y

**Comment by mf-code** [comment]
[6193] - openjdk8 - Allure report on Netlify: https://5f723faa0727d08210682dbd--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[6193] - openjdk10 - Allure report on Netlify: https://5f7240b5c2a68e737916429f--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[6195] - rl8dyYgKijGI

**Comment by mf-code** [comment]
[6195] - 6WupePED9TeL

**Comment by mf-code** [comment]
[6195] - LWSXaCJRgJpg

**Comment by mf-code** [comment]
[6195] - zzG0b5r8vAPK

**Comment by mf-code** [comment]
[6195] - RDcRhBjBHMFX

**Comment by mf-code** [comment]
[6195] - JUwucNF777mv

**Comment by mf-code** [comment]
[6195] - I2Wkx9Zn63OF

**Comment by mf-code** [comment]
[6195] - ul7Y4dlIuBKY

**Comment by mf-code** [comment]
[6195] - KkhCTZfCW5lE

**Comment by mf-code** [comment]
[6195] - TKHkFqtld0nM

**Comment by mf-code** [comment]
[6195] - 4Y63dYpHplGb

**Comment by mf-code** [comment]
[6195] - qeNzMsQS3jno

**Comment by mf-code** [comment]
[6195] - UcNHi6pfsSEY

**Comment by mf-code** [comment]
[6195] - Gf7MIed0fJpg

**Comment by mf-code** [comment]
[6195] - OhuEx2rI5CAW

**Comment by mf-code** [comment]
[6195] - Q7b9dWi5fLT2

**Comment by mf-code** [comment]
[6195] - MpumDnLu792X

**Comment by mf-code** [comment]
[6195] - gwVnw6rOgqUg

**Comment by mf-code** [comment]
[6195] - KIKEGvdf5qry

**Comment by mf-code** [comment]
[6195] - 3avVV9Xc2rtr

**Review by AlexSofft — APPROVED** [review]

**Comment by mf-code** [comment]
[6211] - Zc11hPrHqLIU

**Comment by mf-code** [comment]
[6211] - RleQfUZrbv7i

**Comment by mf-code** [comment]
[6211] - FUxEYo55WP7O

**Comment by mf-code** [comment]
[6211] - 8bjHU7U25tYf

**Comment by mf-code** [comment]
[6211] - Dj7vNtbl1EF1

**Comment by mf-code** [comment]
[6211] - yyRRTFwaKr34

**Comment by mf-code** [comment]
[6211] - 6HQnCVxFV2O8

**Comment by mf-code** [comment]
[6211] - AxwGA8rLI9ja

**Comment by mf-code** [comment]
[6211] - 8TgQLUU9jS8P

**Comment by mf-code** [comment]
[6211] - suc0F5f85tMZ

**Comment by mf-code** [comment]
[6211] - yoCWKDHoBY0N

**Comment by mf-code** [comment]
[6211] - NawLhl1fB6ml

**Comment by mf-code** [comment]
[6211] - yDk8996CW0Ky

**Comment by mf-code** [comment]
[6211] - Kpt4TtvWTRa2

**Comment by mf-code** [comment]
[6211] - BjXV17fc8a3C

**Comment by mf-code** [comment]
[6211] - 0KXOKnWho5Zz

**Comment by mf-code** [comment]
[6211] - CJhfVrpooACc

**Comment by mf-code** [comment]
[6211] - WWOmMnNBWcBB

**Comment by mf-code** [comment]
[6212] - 1R0ALmT7tIUV

**Comment by mf-code** [comment]
[6212] - 6xSXePqMkmyJ

**Comment by mf-code** [comment]
[6212] - mTEjDtWU0mIi

**Comment by mf-code** [comment]
[6212] - QwcYlVgNW8EF

**Comment by mf-code** [comment]
[6212] - 3iclh9QxCUlD

**Comment by mf-code** [comment]
[6212] - dPOLyha1v4F7

**Comment by mf-code** [comment]
[6212] - pCivyejLLsJC

**Comment by mf-code** [comment]
[6212] - DsKrgomqk3fX

**Comment by mf-code** [comment]
[6212] - awVmgq2yEzJC

**Comment by mf-code** [comment]
[6212] - Dnd0pP6uE7UJ

**Comment by mf-code** [comment]
[6212] - mODztGkAV9xP

**Comment by mf-code** [comment]
[6212] - HxYybfTZ4kbT

**Comment by mf-code** [comment]
[6212] - 4FbXnGZRxbfw

**Comment by mf-code** [comment]
[6212] - 8L11wTuMs2qc

**Comment by mf-code** [comment]
[6212] -

**Comment by mf-code** [comment]
[6212] - EencEKlM2yVT

**Comment by mf-code** [comment]
[6212] - Txq1dLkfdRHE

**Comment by mf-code** [comment]
[6212] - 6AZ554Znn0OL

**Comment by mf-code** [comment]
[6222] - OrFoG76Wn9CN

**Comment by mf-code** [comment]
[6222] -

**Comment by mf-code** [comment]
[6222] -

**Comment by mf-code** [comment]
[6222] -

**Comment by mf-code** [comment]
[6222] - uRCvpWHyug43

**Comment by mf-code** [comment]
[6222] - bI8qvwHxkgXA

**Comment by mf-code** [comment]
[6222] - OO6rx83fQmPb

**Comment by mf-code** [comment]
[6222] - Xqwq2WX22WQn

**Comment by mf-code** [comment]
[6222] - jOEJ42sNzgom

**Comment by mf-code** [comment]
[6222] - pSqNMJxrQFIX

**Comment by mf-code** [comment]
[6222] - XjOTklw3hJx5

**Comment by mf-code** [comment]
[6222] - osZbZV40c6Oe

**Comment by mf-code** [comment]
[6222] - B0gdjqxgTp5C

**Comment by mf-code** [comment]
[6222] - SpHA90QYR9Kx

**Comment by mf-code** [comment]
[6222] - 1SfW8G9tMQvr

**Comment by mf-code** [comment]
[6222] - 0xHivJFem2zY

**Comment by mf-code** [comment]
[6222] - OjJ8Gv0HBfAS

**Comment by mf-code** [comment]
[6222] - VYVOR1iGW3rm

**Comment by mf-code** [comment]
[6222] - openjdk8 - Allure report on Netlify: https://5f770a5c8a6fc70e11b622c7--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[6222] - openjdk10 - Allure report on Netlify: https://5f770b36c22cce32d3d84b12--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[6231] - RS7utrW0HHzf

**Comment by mf-code** [comment]
[6231] - 6AgENOUVSr31

**Comment by mf-code** [comment]
[6231] - cOQljQ9zDkjb

**Comment by mf-code** [comment]
[6231] - c9ahwkYCAO7x

**Comment by mf-code** [comment]
[6231] - C4YbFNBILNLE

**Comment by mf-code** [comment]
[6231] - gOgZUo6SgMNs

**Comment by mf-code** [comment]
[6231] - VWWSXHKrOXIM

**Comment by mf-code** [comment]
[6231] - MpDrEy9R5Q69

**Comment by mf-code** [comment]
[6231] - EjYa5K03BZ6f

**Comment by mf-code** [comment]
[6231] - IMVDB6rrD26o

**Comment by mf-code** [comment]
[6231] - d3U1MQ0H866p

**Comment by mf-code** [comment]
[6231] - 5taPd12qnJ1d

**Comment by mf-code** [comment]
[6231] - r1vQZAyfSIMF

**Comment by mf-code** [comment]
[6231] - jj87zqXKyYEb

**Comment by mf-code** [comment]
[6231] - gH6bZwf18A6k

**Comment by mf-code** [comment]
[6231] - GcyEaot62PyM

**Comment by mf-code** [comment]
[6231] - rs37z6MbM5j9

**Comment by mf-code** [comment]
[6231] - ErhtfHmxG9Yr

**Comment by mf-code** [comment]
[6231] - openjdk8 - Allure report on Netlify: https://5f7c4353bd98af635bc04d22--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[6231] - openjdk10 - Allure report on Netlify: https://5f7c44c893b9cd3d4dfbe949--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[6231] - LgAXLwWktZzk

**Comment by mf-code** [comment]
[6231] - ziqKPwCBUCsZ

**Comment by mf-code** [comment]
[6231] - o9QjZ2BY2xup

**Comment by mf-code** [comment]
[6231] - lRin1MIOaG1q

**Comment by mf-code** [comment]
[6231] - b2jXLR52vOd5

**Comment by mf-code** [comment]
[6231] - pICDstatS36U

**Comment by mf-code** [comment]
[6231] - 68W78IMf93Zn

**Comment by mf-code** [comment]
[6231] - B2GtnQ0LsJhr

**Comment by mf-code** [comment]
[6231] - RdJQMEt9s2Mm

**Comment by mf-code** [comment]
[6231] - d4vGQVMA7ShU

**Comment by mf-code** [comment]
[6231] - MABBG61gd68D

**Comment by mf-code** [comment]
[6231] - Rqb8BZwCOznX

**Comment by mf-code** [comment]
[6231] - urfp3IlRRAHv

**Comment by mf-code** [comment]
[6231] - YDgYOkfi9SkD

**Comment by mf-code** [comment]
[6231] - Co59NRnCBwzd

**Comment by mf-code** [comment]
[6231] - ayKrXlw0127Y

**Comment by mf-code** [comment]
[6231] - lGKdBWzf0Ewx

**Comment by mf-code** [comment]
[6231] - 5eEXuWvVE2im

**Comment by mf-code** [comment]
[6231] - openjdk8 - Allure report on Netlify: https://5f7c64019b60aa1d7be3cee1--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[6231] - openjdk10 - Allure report on Netlify: https://5f7c657610416e1b7b843897--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[6235] - T3juAca1XRcs

**Comment by mf-code** [comment]
[6235] - wdBUOGgufEv2

**Comment by mf-code** [comment]
[6235] - w8xHXvjos5BO

**Comment by mf-code** [comment]
[6235] - q0qrNBWRdvdY

**Comment by mf-code** [comment]
[6235] - tjDGv9EQmws9

**Comment by mf-code** [comment]
[6235] - yxXhupceAWD9

**Comment by mf-code** [comment]
[6235] - afNbgFPoo0mD

**Comment by mf-code** [comment]
[6235] - ZWQf13dNjcxB

**Comment by mf-code** [comment]
[6235] - tQbDJF0KOZhR

**Comment by mf-code** [comment]
[6235] - 33x73NiiLcYM

**Comment by mf-code** [comment]
[6235] - zl9IhLvtDQYi

**Comment by mf-code** [comment]
[6235] - MYDaAAI07FbE

**Comment by mf-code** [comment]
[6235] - mJ2zUNTeEWuw

**Comment by mf-code** [comment]
[6235] - 0eihREGAFbbT

**Comment by mf-code** [comment]
[6235] - f8Os3cFPFEfP

**Comment by mf-code** [comment]
[6235] - rHyjZ4xHBGe2

**Comment by mf-code** [comment]
[6235] - VO0JRLysxcup

**Comment by mf-code** [comment]
[6235] - PBqmrw4ZW2X4

**Comment by mf-code** [comment]
[6235] - openjdk8 - Allure report on Netlify: https://5f7dc9508d814119a9dc1104--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[6235] - openjdk10 - Allure report on Netlify: https://5f7dca6485ba53186a2983f4--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[6237] - mBdYOpc9N4SS

**Comment by mf-code** [comment]
[6237] - Zjio5zSf5s1Y

**Comment by mf-code** [comment]
[6237] - UNb9POcq587R

**Comment by mf-code** [comment]
[6237] - HXMK58AvLeu1

**Comment by mf-code** [comment]
[6237] - sepY9d0AJrew

**Comment by mf-code** [comment]
[6237] - ZBC6uhpgfkw7

**Comment by mf-code** [comment]
[6237] - AVIszQAGxCRQ

**Comment by mf-code** [comment]
[6237] - LPD2MNYA4IaH

**Comment by mf-code** [comment]
[6237] - J4b4toYph9uJ

**Comment by mf-code** [comment]
[6237] - u3X3ha5L9f6U

**Comment by mf-code** [comment]
[6237] - CoYtjRXhmqvB

**Comment by mf-code** [comment]
[6237] - uVD1jlA7VV1R

**Comment by mf-code** [comment]
[6237] - gGy0htf1qski

**Comment by mf-code** [comment]
[6237] - 4sbreVhzMWgJ

**Comment by mf-code** [comment]
[6237] - onSBCJy6akx8

**Comment by mf-code** [comment]
[6237] - 1l6Ms7NBRV1N

**Comment by mf-code** [comment]
[6237] - xdkhHAez9eFb

**Comment by mf-code** [comment]
[6237] - VhwZuw8PTs9s

**Comment by mf-code** [comment]
[6237] - openjdk8 - Allure report on Netlify: https://5f7ed5af6d7fa238d4840dc4--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[6237] - openjdk10 - Allure report on Netlify: https://5f7ed75a69758a365e795fef--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/unit/TableUnitTests.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/jdi-testing/jdi-light/pullRequest?prid=6249199&bid=20350762)

**Comment by mf-code** [comment]
[6272] - PgYXBkDPJlJ5

**Comment by mf-code** [comment]
[6272] - j0IwLroUxOFg

**Comment by mf-code** [comment]
[6272] - MDnC0DzQA1VF

**Comment by mf-code** [comment]
[6272] - YCUJNtZ5hnOr

**Comment by mf-code** [comment]
[6272] - 5LgJmKp0RDGA

**Comment by mf-code** [comment]
[6272] - 813lzksH53IN

**Comment by mf-code** [comment]
[6272] - JhS4IWI7WW24

**Comment by mf-code** [comment]
[6272] - svfmso5m0j6L

**Comment by mf-code** [comment]
[6272] - kabFaqmcconJ

**Comment by mf-code** [comment]
[6272] - QjJp78HQQlEF

**Comment by mf-code** [comment]
[6272] - SJtoTKuWA3DM

**Comment by mf-code** [comment]
[6272] - QFwqvMpGN69R

**Comment by mf-code** [comment]
[6272] - PEfDHNlLbUSv

**Comment by mf-code** [comment]
[6272] - qArL9imMjP9B

**Comment by mf-code** [comment]
[6272] - EcVkt1hATqLv

**Comment by mf-code** [comment]
[6272] - i4V9SyEyTqNV

**Comment by mf-code** [comment]
[6272] - ZgxomcMi5Vft

**Comment by mf-code** [comment]
[6272] - h32DZLY7cTUU

**Comment by mf-code** [comment]
[6272] - pWaRVHW1nfcF

**Comment by mf-code** [comment]
[6272] - PYG4E6uE8nYV

**Comment by mf-code** [comment]
[6272] - 1UFBETB3kQ2c

**Comment by mf-code** [comment]
[6272] - uJUZdjO2uY80

**Comment by mf-code** [comment]
[6272] - mYDMRcfchpdC

**Comment by mf-code** [comment]
[6272] - lr2CxdnrF8GP

**Comment by mf-code** [comment]
[6272] - g3lBFaNLdb5y

**Comment by mf-code** [comment]
[6272] - YF0KGqzgiNPF

**Comment by mf-code** [comment]
[6272] - YaKbbQQS1ibM

**Comment by mf-code** [comment]
[6272] - sy0iZLDfCTeB

**Comment by mf-code** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/unit/TableUnitTests.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/jdi-testing/jdi-light/pullRequest?prid=6249199&bid=20350762)

**Comment by mf-code** [comment]
[6279] - osSSO3Uomjge

**Comment by mf-code** [comment]
[6279] - x3eWAA358IV0

**Comment by mf-code** [comment]
[6279] - MSYGlU5w81ZD

**Comment by mf-code** [comment]
[6279] - fbHFV3LgSb8G

**Comment by mf-code** [comment]
[6279] - 1GEXMrTNON1l

**Comment by mf-code** [comment]
[6279] - iy8sV7QrPkQg

**Comment by mf-code** [comment]
[6279] - aSYqguVxgqvr

**Comment by mf-code** [comment]
[6279] - fSc5MqpbaHd6

**Comment by mf-code** [comment]
[6279] - YzSp9JuVabLA

**Comment by mf-code** [comment]
[6279] - NjWtRIjW8271

**Comment by mf-code** [comment]
[6279] - BTycyEkfQB1K

**Comment by mf-code** [comment]
[6279] - ZDp25xhper0Y

**Comment by mf-code** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/unit/TableUnitTests.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/jdi-testing/jdi-light/pullRequest?prid=6249199&bid=20350762)

**Comment by mf-code** [comment]
[6386] - 6Wi9E4ZdHyXc

**Comment by mf-code** [comment]
[6386] - mBcn9jYDxX1s

**Comment by mf-code** [comment]
[6386] - czo0NCfakt9v

**Comment by mf-code** [comment]
[6386] - 9V6myFLMG6HO

**Comment by mf-code** [comment]
[6386] - QAPbzz8N7ChL

**Comment by mf-code** [comment]
[6386] - dBHiFSRktUmL

**Comment by mf-code** [comment]
[6386] - k9bX5PTq5uFG

**Comment by mf-code** [comment]
[6386] - aOQnA73NwXJq

**Comment by mf-code** [comment]
[6386] - iRxMaeu53Jcx

**Comment by mf-code** [comment]
[6386] - kzIplcJvDK3N

**Comment by mf-code** [comment]
[6386] - ZazaHNHaDGuM

**Comment by mf-code** [comment]
[6386] - eAs7E7frknNc

**Comment by mf-code** [comment]
[6386] - t2bFYDzXBYy1

**Comment by mf-code** [comment]
[6386] - jQ0WT6FG42d6

**Comment by RomanIovlev** [comment]
Merged into origin pull_request_for_Roman

**Pull request closed without merging**